trinity/plugins/builtin/tx_pool/pool.py

```diff
@@ -87,6 +87,9 @@ def register_peer(self, peer: BasePeer) -> None:
             if len(filtered_tx) == 0:
                 continue
 
+            if receiving_peer.is_closing:
```

Comment:
is_closing is a method

Reply:
Also, as I mentioned in https://github.com/ethereum/py-evm/issues/1016#issuecomment-405488411 I believe the real issue is in `PeerPool` and once it's fixed there this will not be needed

Reply:
With #1022 this should not be needed because `ConnectedPeersIterator.__anext__()` will perform this exact check for every peer it returns, and here there's nowhere where we yield control between the check in `__anext__()` and this check